src/components/TodoContainer.jsx

```diff
@@ -4,39 +4,104 @@ import TodoList from "./TodoList.jsx";
 import {fetchData} from "../util/fetchData.js";
 import {createData} from "../util/createData.js";
 import {removeData} from "../util/removeData.js";
+import {sortByFieldName} from "../util/sort.js";
+import {parseSortArguments} from "../util/parser.js";
 
 const TodoContainer = () => {
     const [todoList, setTodoList] = useState([]);
     const [isLoading, setIsLoading] = useState(true);
     const [titleError, setTitleError] = useState("")
+    const [sort, setSort] = useState("title-asc")
 
     useEffect(() => {
-        fetchData(setTodoList, setIsLoading);
+        const getData = async function(isAsc, sortFieldName) {
```

Comment:
As your app grows and you incorporate other sources of data the word "data" on `fetchData` and `getData` becomes too ambiguous. User profiles, and calendar integration come to mind. I recommend short and descriptive names.

`getData` does not need any arguments. You can safely reference state variables or other variables inside the useEffect by being aware of lexical scope.